grid/recruitment/list-candidates/maps/greenhouse.test.ts

```diff
@@ -0,0 +1,8 @@
+import {
+  listCandidatesTest,
+} from './list-candidates';
+
+listCandidatesTest('greenhouse', {
+  validJobId: '4031896006',
+  invalidJobId: '5031896006',
```

Comment:
I suggest using an obviously invalid job id, that has no possibility of becoming valid in the future (although I realize it's unlikely)

```suggestion
  invalidJobId: 'INVALID',
```